example/standalone_git/README.md

```diff
@@ -0,0 +1,26 @@
+# NiFi Registry with GIT version control integration
+
+This example deploys a NiFi Registry container with git integration (Github). 
+Before running the example, these variables must be set beforehand. 
+
+| Variable   | Default value | Description |
+| ------------- | ------------- | ------------- |
+| `git_remote_url`  | (empty)  | URL of the remote git repository. Must be HTTPS. |
+| `git_checkout_branch`  | (empty)  |Branch to checkout and track. If none is specified, repository is only cloned and no branch switching is done.|
+| `git_flow_storage_directory`  | /opt/nifi-registry/flow-storage  | Default value is set by image, original default value was "./flow-storage". This variable is used in clone script as a destination directory for clone. It is also used in NiFi Registry configuration.|
+| `git_remote_to_push`  | origin  |This variable is used in clone script to set origin name using `-o, --origin <name> use <name> instead of 'origin' to track upstream`. It is also used in NiFi Registry configuration.|
+| `git_access_user`  | (empty)  |Username|
+| `git_access_password`  | (empty)  |Password|
+| `git_user_name`  | nifi-registry  |Developer|
+| `git_user_email`  | nifi-registry@localhost  |Developer email|
```

Comment:
In our session with Fredrik he said that we don't need these two

Reply:
Yes, thats right. But if we omit the variables, the "commit-owner" will be `nifi` or `nifi-registry`: 
![image](https://user-images.githubusercontent.com/43400021/104178330-6cff2a00-540a-11eb-8f06-ed675b6c2d0f.png)

I think it is better that the developers name is the "commit-owner" rather than `nifi` or `nifi-registry`.  I suggest we let these variables be optionals, maybe this can be solve better.

Reply:
It will be wrong as well when i commit something and you name will be shown as the commiter

Reply:
Anyway, i agree with you

Reply:
> In our session with Fredrik he said that we don't need these two
Are you talking about git access user ?@karima